tests/stressTest2.js

```diff
@@ -0,0 +1,28 @@
+import http from 'k6/http';
+import { sleep } from 'k6';
+
+export let options = {
+  vus: 600,
+  duration: '30s'
+};
+
+export default function() {
```

Comment:
interesting approach

Reply:
this one didn't really work, I think I'll toy with it, but the other worked well anyhow.